packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -352,6 +353,8 @@ module.exports = ctx => ({
         $page.redir = `/docs/guides/${found.guide}/${found.sections[0]}/`
       }
     }
+    
+    frontmatter.canonicalUrl = `https://developer.okta.com${path}`;
```

Comment:
This might work, I can't say if it's a good approach.
Also found this on aws:
https://repost.aws/questions/QUlRZYDOhmTm-PEAyS-I6muQ/how-to-avoid-google-to-index-cloudfront-served-content

Reply:
Yeah I saw this post earlier. I need to discuss this with Doug Chan as it needs some investigation on CF config also. Since he is OOO for a couple of days, I put in this code to help alleviate the problem a bit.